website/blog/2025-04-08-run-ferretdb-postgres-documentdb-extension-cnpg-kubernetes.md

```diff
@@ -0,0 +1,243 @@
+---
+slug: run-ferretdb-postgres-documentdb-extension-cnpg-kubernetes
+title: 'How to deploy FerretDB with CloudNativePG on Kubernetes'
+authors: [alex]
+description: >
+  Learn how to deploy FerretDB with PostgreSQL using CloudNativePG on Kubernetes.
+image: /img/blog/ferretdb-cnpg.jpg
+tags: [compatible applications, tutorial, cloud, postgresql tools, open source]
+---
+
+![Deploy FerretDB with CloudNativePG on Kubernetes](/img/blog/ferretdb-cnpg.jpg)
+
+Running FerretDB on Kubernetes typically requires managing a PostgreSQL backend.
+With [CloudNativePG](https://cloudnative-pg.io/), a Kubernetes-native operator for PostgreSQL, you can automate provisioning, scaling, and failover while ensuring a production-ready infrastructure.
+
+<!--truncate-->
+
+We previously covered [how to run FerretDB on Kubernetes using CloudNativePG (CNPG) as the PostgreSQL operator](https://blog.ferretdb.io/run-ferretdb-cloudnativepg-kubernetes/).
+While that guide remains relevant for FerretDB v1.x, we've since [released FerretDB v2.0](https://blog.ferretdb.io/ferretdb-v2-ga-open-source-mongodb-alternative-ready-for-production/), which brings significant changes: improved performance, better MongoDB compatibility, and new features.
```

Comment:
That wasn't a guide

Reply:
Most technical blog posts are "guides", especially if they're instructionals like this one and the previous blog post. Essentially, it's a guide on how to set up FerretDB with CNPG on Kubernetes.